crnk-client-angular-ngrx/operations/crnk.operations.effects.ts

```diff
@@ -62,9 +62,9 @@ export class OperationsEffects {
 		'Accept': 'application/json-patch+json'
 	});
 
-	@Effect() applyResources$ = this.actions$
-		.ofType(NgrxJsonApiActionTypes.API_APPLY_INIT)
-		.withLatestFrom(this.store, (action, state: any) => {
+	@Effect() applyResources$ = this.actions$.pipe(
```

Comment:
is this sufficient or are there more places left in the project?

Reply:
ping @gianinbasler

Reply:
This is the only place I found in the project which is not compatible with ngrx v7+.